gratipay/billing/payday.py

```diff
@@ -408,23 +408,22 @@ def payout(self):
         log("Starting payout loop.")
         participants = self.db.all("""
             SELECT p.*::participants
-              FROM participants p
-             WHERE balance > 0
-               AND ( SELECT count(*)
-                       FROM exchange_routes r
-                      WHERE r.participant = p.id
-                        AND network = 'balanced-ba'
-                   ) > 0
+              FROM exchange_routes r
+              JOIN participants p ON p.id = r.participant
```

Comment:
If a user had 4 old bank accounts and 1 active one, wouldn't this result in multiple calls to `ach_credit` with the same participant? Maybe changing this to `current_exchange_routes` would solve the problem?